src/index.ts

```diff
@@ -0,0 +1,4 @@
+import { cli, command, error } from './waterfall-cli';
```

Comment:
Let's add some commenting:

```suggestion
// Imports
import { cli, command, error } from './waterfall-cli';

// Exports
```